algobot/interface/configuration.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, parent: QMainWindow, logger: Logger = None):
         }
 
         self.lossTypes = ("Trailing", "Stop")
```

Comment:
we should probably leverage the actual enum values? let's not use integers but actual strings? what do you think? :)

Reply:
Could work - trying to think of how would be best to handle semantics vs presentation

Reply:
Well, as long as the enums stay the same, we can just replace the value.
So TRAILING can change from 1 to "Trailing", then we don't even need a mapper ;p